Sources/Operation.swift

```diff
@@ -36,11 +49,17 @@ struct OperationBuilder: Builder {
     let responses: [Int : Reference<ResponseBuilder>]
     let defaultResponse: Reference<ResponseBuilder>?
     let deprecated: Bool
+    let operationId: String?
+    let tags: [String]?
```

Comment:
non-optional, defaulting to empty array
Let's do the same for security requirements as well.